Cargo.toml

```diff
@@ -45,9 +45,9 @@ version = "=0.1.24"
 default-features = false
 
 [workspace.dependencies.snarkvm]
-#path = "../snarkVM"
-git = "https://github.com/AleoNet/snarkVM.git"
-rev = "be171ce" # If this is updated, the rev in `node/rest/Cargo.toml` must be updated as well.
+git = "https://github.com/joske/snarkVM.git"
```

Comment:
Attention: snarkOS will no longer be using `AleoNet`'s implementation of snarkVM after this PR. Proceed with caution.

Reply:
@joske I'm guessing this was changed for testing purposes and we can change it back to `AleoNet`'s snarkVM ?

Reply:
And merge AleoNet/staging + run `cargo fmt` 🙏

Reply:
Yes, this was because the snarkVM PR needs to be merged first. It was always intended that it be changed to an Aleonet snarkVM repo before merging this.